packages/xsnap/lib/console-shim.js

```diff
@@ -1,16 +1,13 @@
 /* global globalThis */
+
+// We use setQuote() below to break the cycle
+// where SES requires console and console is
+// implemented using assert.quote from SES.
+let quote = _v => '[?]';
+
 function tryPrint(...args) {
-  try {
-    // eslint-disable-next-line
-    print(...args.map(arg => typeof arg === 'symbol' ? arg.toString() : arg));
-  } catch (err) {
-    // eslint-disable-next-line
-    print('cannot print:', err.message);
-    args.forEach((a, i) => {
-      // eslint-disable-next-line
-      print(` ${i}:`, a.toString ? a.toString() : '<no .toString>', typeof a);
-    });
-  }
+  // eslint-disable-next-line
```

Comment:
What are you disabling here?

Reply:
eslint doesn't know that xsnap has `globalThis.print`.

Reply:
`no-undef` was not enough somehow.

note the `eslint-disable-next-line` was there before this change. twice, in fact.

Reply:
Please add a comment explaining this, so it won't look so mysterious to the next reader.

Reply:
`/* global print */` also works, and TSC interprets it as `print : any`.